proposals/NNNN-retroactive-protocol-refinement.md

```diff
@@ -0,0 +1,111 @@
+# Feature name
```

Comment:
Name needs updating :)

Reply:
Thanks, David, I've updated it. Its not like the proposal is going anywhere anyway 🙂